### ISSUE TITLE
Display Form on mobile phones

### ISSUE THREAD

**Issue by ThibautPV** [codeigniter4/shield#271]
There is a problem with the display of forms on cell phones. (see attached images)

Indeed, it should be modified : 
- class="card col-5 shadow-sm"

With: 
- class="card col-12 col-md-5 shadow-sm"

The views "login", "register", "magic_link_message" and "magic_link_form" are concerned

Before :
<img width="310" alt="Capture d’écran 2022-07-02 à 18 21 22" src="https://user-images.githubusercontent.com/75485473/177008576-2f351db7-6109-452c-ace1-6a887bc38bdf.png">

After :
<img width="313" alt="Capture d’écran 2022-07-02 à 18 21 52" src="https://user-images.githubusercontent.com/75485473/177008578-4d7feb2e-0cae-457e-a2af-12d228e5654f.png">

I also wonder if it would be necessary to add a "class="row"" div after the container?

**Comment by MGatner** [comment]
Feel free to send over a PR!

> add a "class="row"" div after the container

What would this accomplish?

**Comment by ThibautPV** [comment]
Well, I tried, and I made a mess of it...

The commit put himself in the other pull request : #272 

See : https://github.com/codeigniter4/shield/compare/develop...ThibautPV:shield:develop

I think I should have "closed" the current pull request before making a new one?

I don't know if there is a solution to fix this.
Delete the fork and re-create it?

I'm not a developer and I don't know how to use Github, I'll have to learn ;)

**Comment by kenjis** [comment]
You can try:
https://github.com/codeigniter4/shield/edit/develop/src/Views/login.php
Select "Create a new branch for this commit and start a pull request"

**Comment by datamweb** [comment]
@ThibautPV  I know starting to learn anything you don't know can be confusing and scary at the beginning
Like you, I struggled a lot to start learning **git**, besides, considering that my mother tongue is not English, learning **git** became many times more difficult for me.
I have now [recorded this 12-minute video](https://aparat.com/v/L80HQ) for you, where you can see how to create **new branch**, **commit** and submit PR.
 Finally, you can create PR related to commit [e70f4f](https://github.com/codeigniter4/shield/commit/e70f4fbd7990da56abc0a1408ebf56b26003f510) after viewing it.
Remember, my goal is to help you get on the path to learning, so you shouldn't depend on me or anyone else.
I hope this video will help you in learning git.

**Comment by kenjis** [comment]
Yes, if you want to learn **git**, learning git commands (and concepts behind them) is the best way.

**Comment by kenjis** [comment]
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#pushing-your-branch

If you have any questions, feel free to ask.

**Comment by datamweb** [comment]
> I have a question though. How can I be sure that I am working on the last version of the file before committing? 

You must update branch `develop ` first before creating a new branch.
You can do it through the following commands:
```
> git switch develop
> git fetch upstream
> git merge upstream/develop
> git push origin develop
```
After updating, create your new branch.
Everything you need is explained [here](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#updating-your-branch).
If you do not understand by reading it, let me know for further guidance.

**Comment by ThibautPV** [comment]
Ok perfect ! 

Thanks for you help
